generator.yml

```diff
@@ -1289,6 +1313,476 @@ builds:
+  - {<< : *arm32_allyes,      << : *mainline,         << : *llvm_full,       boot: false, << : *llvm_16}
+  - {<< : *arm32_alpine,      << : *mainline,         << : *llvm_full,       boot: true,  << : *llvm_16}
+  - {<< : *arm32_fedora,      << : *mainline,         << : *llvm_full,       boot: true,  << : *llvm_16}
+  - {<< : *arm32_suse,        << : *mainline,         << : *llvm_full,       boot: true,  << : *llvm_16}
+  - {<< : *arm64,             << : *mainline,         << : *llvm_full,       boot: true,  << : *llvm_16}
+  - {<< : *arm64be,           << : *mainline,         << : *llvm_full,       boot: true,  << : *llvm_16}
+  - {<< : *arm64_lto_full,    << : *mainline,         << : *llvm_full,       boot: true,  << : *llvm_16}
+  - {<< : *arm64_lto_thin,    << : *mainline,         << : *llvm_full,       boot: true,  << : *llvm_16}
+  - {<< : *arm64_cfi,         << : *mainline,         << : *llvm_full,       boot: true,  << : *llvm_16}
+  - {<< : *arm64_cfi_lto,     << : *mainline,         << : *llvm_full,       boot: true,  << : *llvm_16}
+  - {<< : *arm64_kasan,       << : *mainline,         << : *llvm_full,       boot: true,  << : *llvm_16}
+  - {<< : *arm64_kasan_sw,    << : *mainline,         << : *llvm_full,       boot: true,  << : *llvm_16}
+  - {<< : *arm64_ubsan,       << : *mainline,         << : *llvm_full,       boot: true,  << : *llvm_16}
+  - {<< : *arm64_allmod,      << : *mainline,         << : *llvm_full,       boot: false, << : *llvm_16}
+  - {<< : *arm64_allmod_lto,  << : *mainline,         << : *llvm_full,       boot: false, << : *llvm_16}
+  - {<< : *arm64_allno,       << : *mainline,         << : *llvm_full,       boot: false, << : *llvm_16}
+  - {<< : *arm64_allyes,      << : *mainline,         << : *llvm_full,       boot: false, << : *llvm_16}
+  - {<< : *arm64_alpine,      << : *mainline,         << : *llvm_full,       boot: true,  << : *llvm_16}
+  - {<< : *arm64_fedora,      << : *mainline,         << : *llvm_full,       boot: true,  << : *llvm_16}
+  - {<< : *arm64_suse,        << : *mainline,         << : *llvm_full,       boot: true,  << : *llvm_16}
+  - {<< : *hexagon,           << : *mainline,         << : *llvm_full,       boot: false, << : *llvm_16}
+  - {<< : *hexagon_allmod,    << : *mainline,         << : *llvm_full,       boot: false, << : *llvm_16}
+  - {<< : *i386,              << : *mainline,         << : *llvm_full,       boot: true,  << : *llvm_16}
+  - {<< : *i386_suse,         << : *mainline,         << : *llvm_full,       boot: false, << : *llvm_16}
+  - {<< : *mips,              << : *mainline,         << : *llvm_full,       boot: true,  << : *llvm_16}
+  - {<< : *mipsel,            << : *mainline,         << : *llvm_full,       boot: true,  << : *llvm_16}
+  # ppc32: Build disabled (https://github.com/ClangBuiltLinux/linux/issues/1679)
+  # - {<< : *ppc32,             << : *mainline,         << : *llvm,            boot: true,  << : *llvm_16}
```

Comment:
https://github.com/ClangBuiltLinux/linux/issues/1679 is marked as fixed in llvm-16. Time to re-enable for for this PR?

Reply:
Sure, although I will have to do `boot: false` due to https://github.com/ClangBuiltLinux/linux/issues/1814. I'll add a patch on top.

Reply:
cfa47bbc73097dfb40f17d68fbf5473652d2acb5